### PR TITLE
display daemon set, get dynamic selectors, get live names, need to work on setting offline/online podlabel/containerIDs

### PR DESCRIPTION
**Pull request by joelparkk**

### CODE REVIEW

File: src/scss/GraphStyles.ts [hunk 1]
```diff
@@ -29,10 +29,30 @@ export let GraphStyles:Cytoscape.Stylesheet[] = [
...
+      // "border-color": "none",
+      // "border-width": "2",
+      color: "whitesmoke",
+    },
+  },
   {
     selector: 'node[class = "pod"]',
     style: {
-      shape: "ellipse",
+      shape: "elipse",
```
Comment (brkg) [inline src/scss/GraphStyles.ts]: is this mispelled?

File: src/components/views/NodeView.tsx [hunk 2]
```diff
@@ -156,6 +183,7 @@ function NodeView(props: any) {
         setTarget(event.target._private.data.label.split(":")[0]);
         setImage(event.target._private.data.id.slice(0,event.target._private.data.id.length - 2));
       }
+      console.log('hemmie',event.target._private.data.id)
```
Comment (brkg) [inline src/components/views/NodeView.tsx]: is this console.log necessary?

File: src/component_data/kDataParser.ts [hunk 3]
```diff
@@ -64,29 +62,31 @@ export function parseData(relevantData: kObjects.anyObject[]): kObject[] {
...
+          ele.spec.type,
         );
         kObjArray.push(newkSerivce);
-        // console.log(kObjArray)
+      } else if(ele.kind === "DaemonSet"){
+        const newkDaemonSet = new kObjects.kDaemonSet(
+          ele.metadata.name,
+          ele.metadata.namespace ? ele.metadata.namespace : "default",
+          ele.kind,
+          //want more info?
```
Comment (brkg) [inline src/component_data/kDataParser.ts:87]: is this comment necessary?

**Comment by joelparkk** [comment]
#76

**Review by brkg — APPROVED** [review]

File: src/scss/GraphStyles.ts (hunk 1 above)
Comment (joelparkk) [reply src/scss/GraphStyles.ts]: yes, fixed thank you

File: src/components/views/NodeView.tsx (hunk 2 above)
Comment (joelparkk) [reply src/components/views/NodeView.tsx]: this console log is for the on click, only necessary for development reasons. Will remove now.

**Review by brkg — APPROVED** [review]

**Pull request merged**